controllers/chefs/getChefs/index.js

```diff
@@ -1,20 +1,44 @@
+import { workStatus } from '#constants/workStatus.js';
 import { getRating } from '#helpers/getRating.js';
 import Chef from '../../../models/chef/Chef.model.js';
 
 export const getChefs = async (req, res) => {
-  const query = {};
+  let query = Chef.find().populate('userId', 'firstName lastName');
+
   if (req.query.isAvailable) {
-    query.isAvailable = req.query.isAvailable === 'true';
+    query.isAvailable = req.query.isAvailable === workStatus.ACTIVE;
   }
-  const chefs = await Chef.find(query).exec();
+  const chefs = await query.exec();
+  const chefsList = [];
+  if (req.query.name) {
```

Comment:
If I don't send name I get empty array chefsList.
---
I think we need do little changes 
 **let** chefsList = [];
  if (req.query.name) {
    const regex = new RegExp(req.query.name, 'i');

    for (const chef of chefs) {
      if ((chef.userId.firstName + chef.userId.lastName).match(regex)) {
        chefsList.push(chef);
      }
    }
  } **else {
    chefsList = [...chefs];
  }**

Reply:
Thanks, I fixed